src/actions/LoginActions.tsx

```diff
@@ -305,45 +295,67 @@ async function safeCreateWallet(account: EdgeAccount, walletType: string, wallet
   return wallet
 }
 
-/**
- * Creates the custom default wallets inside a new account.
- * The `currencyCodes` are in the format "ETH:DAI",
- * so we may need to enable tokens on some of the created wallets.
- */
-async function createCustomWallets(account: EdgeAccount, fiatCurrencyCode: string, currencyCodes: string[], dispatch: Dispatch) {
-  const currencyInfos: EdgeCurrencyInfo[] = []
+// The `currencyCodes` are in the format "ETH:DAI",
+const currencyCodesToEdgeTokenIds = (account: EdgeAccount, currencyCodes: string[]): EdgeTokenId[] => {
```

Comment:
We already have a `upgradeCurrencyCodes` helper in WalletListModal which is extremely similar to this, but not exactly the same. I wonder if we should make a `src/util/currencyCodes.ts` file to gather these conversion routines together into one place, and document the differences.

Your actual `currencyCodesToEdgeTokenIds` implementation seems great - I just don't think this is the right place for it.

Reply:
I thought there was but didn't look hard enough to find it I guess. I like the idea of consolidating but will save it for a future task